### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 106) 

### LINKED ISSUE #3735: [devtools-translate] Spanish - What's New in DevTools (Chrome 106)
Post: https://developer.chrome.com/blog/new-in-devtools-106/
Translated by: Sep 23

Translator: @Caballerog
Reviewers: @and-oli @midudev

### PR DESCRIPTION
**Pull request by Caballerog**
feat(devtools): translate Spanish - WNDT (Chrome 106) close #3735

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks @Caballerog! Over to @and-oli  and @midudev for review!

File: site/es/blog/new-in-devtools-106/index.md [hunk 1]
```diff
@@ -59,76 +70,98 @@ async increment() {
...
 
 
 <!-- ## Improved call stack during debugging  {: #call-stack } -->
+## Pila de llamadas mejorada durante la depuración {: #call-stack }
 
 <!-- With the **Automatically add known third-party scripts to ignore list** setting, the call stack now shows only frames that are relevant to your code. -->
+Con la configuración **Agregar automáticamente scripts de terceros conocidos para ignorar la lista**, la pila de llamadas ahora muestra solo los marcos que son relevantes para su código.
 
 <!-- Open this [demo](https://ng-devtools.netlify.app/) and set a breakpoint at the `increment()` function in `app.component.ts`. Click the increment button on the page to trigger the breakpoint. The call stack shows only frames from your code (e.g.  `app.component.ts` and `button.component.ts`).  -->
+Abra esta [demo](https://ng-devtools.netlify.app/) y establezca un punto de interrupción en la función `increment()` en `app.component.ts`. Haga clic en el botón de `increment` en la página para activar el punto de interrupción. La pila de llamadas muestra solo frameworks de su código (por ejemplo, `app.component.ts` y `button.component.ts`).
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:127]:
```suggestion
Abra esta [demo](https://ng-devtools.netlify.app/) y establezca un punto de interrupción en la función `increment()` en `app.component.ts`. Haga clic en el botón de `increment` en la página para activar el punto de interrupción. La pila de llamadas muestra solo marcos de su código (por ejemplo, `app.component.ts` y `button.component.ts`).
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 2]
```diff
@@ -59,76 +70,98 @@ async increment() {
...
 
 <!-- To view all frames, enable **Show ignore-listed frames**. Previously, DevTools displayed all frames by default.  -->
+Para ver todos los fotogramas, habilite **Mostrar fotogramas ignorados**. Anteriormente, DevTools mostraba todos los fotogramas de forma predeterminada.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:130]:
```suggestion
Para ver todos los marcos de pila, habilite **Mostrar marcos de la lista de ignorados**. Anteriormente, DevTools mostraba todos los marcos de forma predeterminada.
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 3]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LpHJbSGra2ZCHpy3ns7q.png", alt="Interactions track in the Performance panel", width="800", height="489" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LpHJbSGra2ZCHpy3ns7q.png", alt="Pista de interacciones en el panel Rendimiento", width="800", height="489" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/6d97228951a6c8884b3ac4b712e966e79f2bdc3c #}
 
 Chromium bug: [1347390](https://crbug.com/1347390)
 
 
 <!-- ## LCP timings breakdown in the Performance Insights panel {: #insights } -->
+## Desglose de tiempos de LCP en el panel de pespectiva de rendimiento {: #insights }
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:189]:
```suggestion
## Desglose de tiempos de LCP en el panel de Performance Insights {: #insights }
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 4]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
 
 <!-- The **Performance Insights** panel now shows the [timings breakdown](web.dev/optimize-lcp/#lcp-breakdown)  of the [Largest Containful Paint (LCP)](/docs/devtools/performance-insights/#largest-contentful-paint). Use these timings information to understand and identify an opportunity to improve LCP performance. -->
+El panel **Perspectiva de Rendimiento** ahora muestra el [desglose de tiempos](web.dev/optimize-lcp/#lcp-breakdown) de la [`Largest Containful Paint` (LCP)](/docs/devtools/performance-insights/#mayor-contenido-pintura). Utilice esta información de tiempos para comprender e identificar una oportunidad para mejorar el rendimiento de LCP.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:192]:
```suggestion
El panel **Performance Insights** ahora muestra el [desglose de tiempos](web.dev/optimize-lcp/#lcp-breakdown) de la [`Largest Containful Paint` (LCP)](/docs/devtools/performance-insights/#mayor-contenido-pintura). Utilice esta información de tiempos para comprender e identificar una oportunidad para mejorar el rendimiento de LCP.
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 5]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hU6RmoRjFskL8P2ZAB9l.png", alt="Desglose de tiempos de LCP en el panel Perspectiva de Rendimiento", width="800", height="523" %}
 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/883542a3727a5bc1415ffee7c7bc7f7218d9e2a5 #}
 
 Chromium bug: [1351735](https://crbug.com/1351735)
 
 
 <!-- ## Auto-generate default name for recordings in the Recorder panel {: #recorder } -->
+## Generar automáticamente el nombre predeterminado para las grabaciones en el panel Grabadora {: #recorder }
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:204]:
```suggestion
## Generar automáticamente el nombre predeterminado para las grabaciones en el panel Recorder {: #recorder }
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 6]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
+
 
 <!-- The **Recorder** panel now automatically generates a name for new recordings. -->
+El panel **Grabadora** ahora genera automáticamente un nombre para las nuevas grabaciones.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:208]:
```suggestion
El panel **Recorder** ahora genera automáticamente un nombre para las nuevas grabaciones.
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 7]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0TMJgVqyk7AeoWIR6Vee.png", alt="Default name for recordings in the Recorder panel", width="800", height="565" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0TMJgVqyk7AeoWIR6Vee.png", alt="Nombre predeterminado para las grabaciones en el panel Grabadora", width="800", height="565" %}
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:210]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0TMJgVqyk7AeoWIR6Vee.png", alt="Nombre predeterminado para las grabaciones en el panel Recorder", width="800", height="565" %}
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 8]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/fbf1466b00d1ff2c36fce81fde1b21f33b689a76 #}
 
 Chromium bug: [1351383](https://crbug.com/1351383)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Destacados varios {: #misc }
 
 <!-- - Previously, [Recorder extensions](/docs/devtools/recorder/reference/#extension-troubleshooting) don’t show up in the **Recorder** panel from time to time. ([1351416](https://crbug.com/1351416)) -->
+- Anteriormente, [Extensiones de grabadora](/docs/devtools/recorder/reference/#extension-troubleshooting) no aparecían en el panel de **Grabadora** de vez en cuando. ([1351416](https://crbug.com/1351416))
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:221]:
```suggestion
- Anteriormente, [Las extensiones para  el panel **Recorder**](/docs/devtools/recorder/reference/#extension-troubleshooting) no aparecían en el panel de vez en cuando. ([1351416](https://crbug.com/1351416))
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 9]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
+
 <!-- - The **Styles** pane now displays a color picker for the [SVG `<stop>`](https://developer.mozilla.org/docs/Web/SVG/Element/stop) element’s `stop-color` property. ([1351096](https://crbug.com/1351096)) -->
+- El panel **Estilos** ahora muestra un selector de color para el `stop-color` del elemento [SVG `<stop>`](https://developer.mozilla.org/docs/Web/SVG/Element/stop) propiedad. ([1351096](https://crbug.com/1351096))
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:224]:
```suggestion
- El panel **Estilos** ahora muestra un selector de color para la propiedad `stop-color` del elemento [SVG `<stop>`](https://developer.mozilla.org/docs/Web/SVG/Element/stop) . ([1351096](https://crbug.com/1351096))
```

File: site/es/blog/new-in-devtools-106/index.md [hunk 10]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
+
 <!-- - Identify script causing [layout](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) as the potential root causes for layout shifts in the **Performance Insights** panel. ([1343019](https://crbug.com/1343019)) -->
+- Identifique el script que está provocando que el [diseño](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) como las posibles causas fundamentales de los cambios de diseño en el panel **Perspectivas de rendimiento**. ([1343019](https://crbug.com/1343019))
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:227]: Let's tweak the wording here a bit to make this clearer + use the English name of the Performance Insights panel.
```suggestion
- Identifique el script que está provocando la ejecución de [layout](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) como una posible causa fundamental de cambios de diseño con el panel **Performance Insights**. ([1343019](https://crbug.com/1343019))
 ```

File: site/es/blog/new-in-devtools-106/index.md [hunk 11]
```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
...
+
 <!-- - Display critical path for LCP web fonts in the **Performance Insights** panel. ([1350390](https://crbug.com/1350390)) -->
+- Muestre la ruta crítica para las fuentes web LCP en el panel **Perpectiva de Rendimiento**. ([1350390](https://crbug.com/1350390))
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-106/index.md:230]:
```suggestion
- Se muestra la ruta crítica para las fuentes web LCP en el panel **Performance Insights**. ([1350390](https://crbug.com/1350390))
```

**Review by and-oli — COMMENTED** [review]
LGTM per some comments I left.

Muchas gracias por la traducción, Carlos!

**Comment by jecfish** [comment]
Reminder to @midudev for review!

**Review by jecfish — APPROVED** [review]

**Pull request merged**